scripts/tests/chiptest/__init__.py

```diff
@@ -175,6 +175,7 @@ def _GetChipReplUnsupportedTests() -> Set[str]:
         "Test_TC_ACE_1_6.yaml",    # Test fails only in chip-repl: Refer--> https://github.com/project-chip/connectedhomeip/pull/27910#issuecomment-1632485584
         "Test_TC_IDM_1_2.yaml",              # chip-repl does not support AnyCommands (19/07/2023)
         "TestGroupKeyManagementCluster.yaml",  # chip-repl does not support EqualityCommands (2023-08-04)
+        "TestIcdManagementCluster.yaml"    # TODO(#30430): add ICD registration support in chip-repl
```

Comment:
Should all the ICD tests be added?

Reply:
No, Only ICD tests with reading RegisteredClients will fail in repl tests.